core/src/test/scala/io/snappydata/CommandLineToolsSuite.scala

```diff
@@ -253,8 +253,8 @@ class CommandLineToolsSuite extends SnappyTestRunner {
         "Option -dir not specified with a value")
 
       consoleOutput = (snappyProductDir +
-          "/sbin/snappy-lead.sh start -locators=localhost:10334 -client-port=2002" +
-          "-dir=/does/not/exist").!!
+          "/sbin/snappy-lead.sh start -locators=localhost:10334 -client-port=2002 -dir= " +
```

Comment:
There should not be a space after '-dir='. The case fails as expected but for a different reason.